lumicks/pylake/fitting/models.py

```diff
@@ -259,6 +259,43 @@ def odijk(name):
     )
 
 
+def dsdna(name, kilobase_pairs, kilobase_pairs_per_micron=0.34, temperature=24.53608821):
+    """Odijk's Extensible Worm-Like Chain model with distance as the dependent variable, using
+    user-specified kilobase-pairs (useful for 10 pN < F < 30 pN).
+
+    References:
+      1. T. Odijk, Stiff Chains and Filaments under Tension, Macromolecules
+         28, 7016-7018 (1995).
+      2. M. D. Wang, H. Yin, R. Landick, J. Gelles, S. M. Block, Stretching
+         DNA with optical tweezers., Biophysical journal 72, 1335-46 (1997).
+
+    Parameters
+    ----------
+    name : str
+        Name for the model. This name will be prefixed to the model parameter names.
+    kilobase_pairs : integer
+        The number of kilobase-pairs in the sample
+    kilobase_pairs_per_micron: float
+        The number of kilobase-pairs evaluating to 1 micron. This is used to convert the kilobase-pairs to microns for the fit function.
+    temperature: float
+        The temperature in celsius. This is used to calculate the boltzmann * temperature value
+    """
+    import scipy.constants as spy
+
+    model = odijk(name)
+    model.defaults[f"{name}/Lc"].value = kilobase_pairs * kilobase_pairs_per_micron
+    # Based on lambda dna C-trap experiments
+    model.defaults[f"{name}/Lp"].value = 56.0
```

Comment:
I'd really prefer a citation over "# Based on lambda dna C-trap experiments" since our users can't really source that.

Reply:
Indeed. I have managed to obtain more citation data, will update as we go